torchci/components/metrics/panels/ScalarPanel.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * A metrics panel that shows a single scalar value.
+ */
+
+import { RocksetParam } from "lib/rockset";
+import { Box, Paper, Typography, Skeleton } from "@mui/material";
+import useSWR from "swr";
+
+const fetcher = (url: string) => fetch(url).then((res) => res.json());
+
+export default function ScalarPanel({
+  // Human-readable title of the panel.
+  title,
+  // Query lambda name in Rockset, ("metrics" collection is assumed).
```

Comment:
Can we make the collection optional and just have it be a default param of metrics?